<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAddBook</class>
 <widget class="QWidget" name="FormAddBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbCota">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(*) Cota:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbAutor">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(*)Autor: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMateria">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(*)Materia:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTitulo">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(*)Titulo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbEditorial">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Editorial:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbAnhoPublicacion">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>AÃ±o Public.:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLugar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Lugar:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbAdquisicion">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>105</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Adquisicion: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbVolumen">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tomo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbEstado">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboEstado">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboAdquisicion">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCota">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAutor">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTitulo">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leMateria">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leEditorial">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAnhoPublicacion">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLugar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leVolumen">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
